src/main/java/org/embulk/input/marketo/delegate/ActivityBulkExtractInputPlugin.java

```diff
@@ -0,0 +1,67 @@
+package org.embulk.input.marketo.delegate;
+
+import org.embulk.base.restclient.ServiceResponseMapper;
+import org.embulk.base.restclient.jackson.JacksonServiceResponseMapper;
+import org.embulk.base.restclient.record.ValueLocator;
+import org.embulk.input.marketo.MarketoService;
+import org.embulk.input.marketo.MarketoServiceImpl;
+import org.embulk.input.marketo.MarketoUtils;
+import org.embulk.input.marketo.rest.MarketoRestClient;
+import org.embulk.spi.DataException;
+import org.embulk.spi.Exec;
+import org.embulk.spi.Schema;
+import org.embulk.spi.type.Types;
+import org.slf4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.Date;
+
+/**
+ * Created by tai.khuu on 9/18/17.
+ */
+public class ActivityBulkExtractInputPlugin extends MarketoBaseBulkExtractInputPlugin<ActivityBulkExtractInputPlugin.PluginTask>
+{
+    private static final Logger LOGGER = Exec.getLogger(ActivityBulkExtractInputPlugin.class);
+
+    public interface PluginTask extends MarketoBaseBulkExtractInputPlugin.PluginTask {}
+
+    public ActivityBulkExtractInputPlugin()
+    {
+        super("activityDate", "marketoGUID");
+    }
+
+    @Override
+    protected InputStream getExtractedStream(PluginTask task, Schema schema)
+    {
+        try (MarketoRestClient marketoRestClient = createMarketoRestClient(task)) {
+            MarketoService marketoService = new MarketoServiceImpl(marketoRestClient);
+            {
```

Comment:
These lines can be compacted by removing some brackets & bring the `catch` out

Reply:
Fixed